react/src/components/ServiceLauncherPageContent.tsx

```diff
@@ -178,6 +192,29 @@ const ServiceLauncherPageContent: React.FC<ServiceLauncherPageContentProps> = ({
     endpointFrgmt,
   );
 
+  const { data: availableRuntimes } = useTanQuery<{
+    runtimes: { name: string; human_readable_name: string }[];
+  }>({
+    queryKey: ['baiClient.modelService.runtime.list'],
+    queryFn: () => {
+      return baiClient.isManagerVersionCompatibleWith('24.03.5')
+        ? baiRequestWithPromise({
+            method: 'GET',
+            url: `/services/_/runtimes`,
+          })
+        : Promise.resolve({
+            runtimes: [
+              {
+                name: 'custom',
+                human_readable_name: 'Custom (Model Definition)',
```

Comment:
Should it be `Custom (Default)`?
```suggestion
                human_readable_name: 'Custom (Default)',
```